backend/fms_core/serializers.py

```diff
@@ -557,11 +558,99 @@ class Meta:
             'parent_sample_id', 
             'parent_sample_name', 
             'platform',
+            'pool_id',
             'project_id', 
             'project_name', 
-            'sample_kind'
+            'sample_kind',
             'strandedness',
             'volume_ratio', 
             ]
 
+class PooledSampleExportSerializer(serializers.Serializer):
+    ''' Serializes a DerivedBySample object, representing a pooled sample, for export to CSV.
+    '''
+    volume_ratio = serializers.DecimalField(max_digits=20, decimal_places=3, read_only=True)
+
+    # Associated project info
+    project_id = serializers.IntegerField(read_only=True, source='derived_sample.project.id')
+    project_name = serializers.CharField(read_only=True, source='derived_sample.project.name')
+    
+    # Sample info
+    alias = serializers.CharField(read_only=True, source='derived_sample.biosample.alias')
+    collection_site = serializers.CharField(read_only=True, source='derived_sample.biosample.collection_site')
+    experimental_groups = serializers.JSONField(read_only=True, source='derived_sample.experimental_group')
+    parent_sample_id = serializers.CharField(read_only=True)
+    parent_sample_name = serializers.CharField(read_only=True)
+    sample_kind = serializers.CharField(read_only=True, source='derived_sample.sample_kind.name')
+
+    # Individual info
+    individual_name = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.name')
+    taxon_id = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.taxon.ncbi_id')
+    taxon_name = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.taxon.name')
+    sex = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.sex')
+    mother = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.mother.name')
+    father = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.father.name')
+    pedigree = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.pedigree')
+    cohort = serializers.CharField(read_only=True, source='derived_sample.biosample.individual.cohort')
+
+    # Library info
+    index = serializers.CharField(read_only=True, source='derived_sample.library.index.name')
+    index_structure = serializers.CharField(read_only=True, source='derived_sample.library.index.structure.name')
+    index_set = serializers.CharField(read_only=True, source='derived_sample.library.index.index_set.name')
+    # sequences_3prime = serializers.CharField(read_only=True, source='derived_sample.library.index.list_3prime_sequences')
+    # sequences_5prime = serializers.CharField(read_only=True, source='derived_sample.library.index.list_5prime_sequences')
+    sequences_3prime = serializers.SerializerMethodField()
```

Comment:
Since these will be the headers i would add index_ before the 2 sequences fields. This could clarify what those sequences are.